lib/utils/AuthContext.js

```diff
@@ -40,6 +40,36 @@ const accessTokenAllowed = {
 	},
 	ws: [],
 };
+
+const tokenAllowed = {
+	http: {
+		[endpoints.session]: ['POST'],
+		[endpoints.testPackGenTokens]: ['POST'],
+		[endpoints.apps]: ['GET'],
+		[endpoints.appById]: ['GET'],
+		[endpoints.appTags]: ['GET'],
+		[endpoints.appConfigs]: ['GET'],
+		[endpoints.appConfigById]: ['GET'],
+		[endpoints.appTestPacks]: ['GET'],
+		[endpoints.appTestPackById]: ['GET'],
+		[endpoints.appTestDefinitions]: ['GET'],
+		[endpoints.appTestDefinitionById]: ['GET'],
+		[endpoints.devices]: ['GET'],
+		[endpoints.testRun]: ['POST'],
+		[endpoints.testRunById]: ['GET', 'DELETE'],
+		[endpoints.testRunOnDevice]: ['DELETE'],
+		[endpoints.testPackRun]: ['POST'],
+		[endpoints.testPackRunById]: ['GET', 'DELETE'],
```

Comment:
Are `testPackRunById`, `testPackRun`, `appTestPacks`, `appTestPackById` make sense in new mode?